ncclient/transport/ssh.py

```diff
@@ -286,17 +290,29 @@ def close(self):
 
 
     # REMEMBER to update transport.rst if sig. changes, since it is hardcoded there
-    def connect(self, host, port=830, timeout=None, unknown_host_cb=default_unknown_host_cb,
-                username=None, password=None, key_filename=None, allow_agent=True,
-                hostkey_verify=True, look_for_keys=True, ssh_config=None, sock_fd=None):
+    def connect(
+            self,
+            host,
+            port                = PORT_NETCONF_DEFAULT,
+            timeout             = None,
+            unknown_host_cb     = default_unknown_host_cb,
+            username            = None,
+            password            = None,
+            key_filename        = None,
+            allow_agent         = True,
+            hostkey_verify      = True,
+            hostkey             = None,
+            look_for_keys       = True,
+            ssh_config          = None,
+            sock_fd             = None):
 
         """Connect via SSH and initialize the NETCONF session. First attempts the publickey authentication method and then password authentication.
 
         To disable attempting publickey authentication altogether, call with *allow_agent* and *look_for_keys* as `False`.
 
         *host* is the hostname or IP address to connect to
 
-        *port* is by default 830, but some devices use the default SSH port of 22 so this may need to be specified
+        *port* is by default PORT_NETCONF_DEFAULT, but some devices use the default SSH port of 22 (PORT_SSH_DEFAULT) so this may need to be specified
```

Comment:
Personally I'm not a fan of this change - it takes something clear and makes it less so (now I have to look elsewhere to find out what PORT_NETCONF_DEFAULT is).

Reply:
It's generally good practice to avoid magic numbers. You're welcome to keep (can revert) but I recommend avoiding